src/components/taskDetails/TaskDependency.tsx

```diff
@@ -0,0 +1,113 @@
+import Link from 'next/link';
+import { ChangeEvent, FC, useState } from 'react';
+import classNames from './task-details.module.scss';
+import { dependency } from '@/app/services/taskDetailsApi';
+import { Textarea } from '.';
+
+type Props = {
+    loading: boolean;
+    fetching: boolean;
+    error: boolean;
+    dependencyData: dependency | undefined;
+    navigateToTask: (taskId: string) => void;
+    isEditing: boolean;
+    updatedDependencies: string[];
+    handleChange: (
+        event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+    ) => void;
+};
+
+const TaskDependency: FC<Props> = ({
+    loading,
+    fetching,
+    error,
+    dependencyData,
+    navigateToTask,
+    isEditing,
+    updatedDependencies,
+    handleChange,
```

Comment:
Is it possible to somehow reduce the number of props passed?

Reply:
For e.g. `navigateToTask` I believe is not being used anywhere in the parent component, so the function can be created within this component.

Reply:
done